src/web_app_skeleton/src/actions/home/index.cr.ecr

```diff
@@ -1,5 +1,9 @@
 <%- if api_only? -%>
 class Home::Index < ApiAction
+  <%- if generate_auth? -%>
+  include Auth::SkipRequireSignIn
```

Comment:
Should this be using the `Api` version?

Reply:
I guess it doesn't matter actually

Reply:
You're right, I'm not sure since this action is at the `/` route and not at `/api/`. If you want I can remove the api version and just leave the lucky welcome page.